source/objects/waveClass.m

```diff
@@ -550,12 +550,16 @@ function plotElevation(obj,rampTime)
                 rampTime double {mustBeReal, mustBeNonNan, mustBeFinite} = 0
             end
 
+            waveElevationSum = zeros(size(obj(1).waveAmpTime(:,2)));

```

Comment:
This works but is quite surprising MATLAB behavior to me. I would have thought that calling an object's method on an array of objects would error, not result in the entire object array being available in the method.